src/main/webapp/creatives/maps/simid_map.css

```diff
@@ -33,23 +34,42 @@ button:hover {
 }
 
 #travelMethod {
-    top: 5px;
+    padding: 4px;
+    background: rgba(0,0,0,0.5);
+    border: 1px solid rgba(255,255,255,0.5);
+    color: white;
+    font-size: 18px;
+    margin-left: 11px;
+    top: 60px;
+    position: absolute;
+}
+
+#timeDisplay {
```

Comment:
looks like the box is always positioned to the same spot? If so, what happens when the line connecting the two places passes through that? Might look a little weird.

Reply:
the line connecting places is centered in the map (it would never be shifted) so this shouldn't be an issue

Reply:
Kaleab is right-- when I run the code with the design changes the box covers up the route. https://screenshot.googleplex.com/GsHaQXSng0Z, https://screenshot.googleplex.com/CjZDko65jUL. Maybe we could move the travel time box towards the travel method dropdown or I can remove some of the text to make it smaller.

Reply:
Hmm is there a way to use the api to know where the route is being placed? Can you then use that to move the box around dynamically?

Reply:
tried to find a way but can't seem to find one using the api

Reply:
Hmm, it is somewhat transparant and they can move the map so probably not a huge deal, but up to you all where you think the best place to put it is so that its as unobtrusive as possible

Reply:
Right I made it transparent for that reason

Reply:
Kristen is going to shorten the amount of words which should help so this design seems okay